iothub/device/devdoc/Convention-based operations.md

```diff
@@ -1,8 +1,8 @@
-## Plug and Play convention compatible APIs
+# Plug and Play convention compatible APIs
 
-#### Common
+## Common
 
-```diff
+```csharp
```

Comment:
we're going to want to keep this block as `diff` so that it highlights the 2 APIs added to ClientOptions

Reply:
what does it do for markdown?

Reply:
Oh, fascinating. I didn't know that. Thanks!